lib/perl/Genome/Process/Command/Run.pm

```diff
@@ -148,7 +148,7 @@ sub _bsub_in_pend_state {
         log_file => $self->workflow_process_out_log,
         hold_job => 1,
         project => $self->process->workflow_name,
-        queue => $ENV{GENOME_LSF_QUEUE_BUILD_WORKFLOW},
+        queue => $ENV{GENOME_LSF_QUEUE_DV2_WORKFLOW},
```

Comment:
I would make `Genome::Process::Command::Run` an exception to this change.  `Process`es and `Build`s are nearly identical in terms of running workflows (and I still hope they are eventually deduplicated!).